### DISCUSSION TITLE
where's the tailwind class for `inherit` or `initial` styles?

### DISCUSSION THREAD

**Discussion by GorvGoyl** [tailwindlabs/tailwindcss#3754] [Help]
where's the tailwind class for `overflow:inherit` or `overflow:initial`? Use case is I want to add `overflow: auto;` only for `sm` screens and the larger screens should have `overflow:inherit` or `overflow:initial` but there are no tailwind class for it. How do people solve this requirement?

**Comment by simonswiss** [comment]
Hey!

These two values do not exist in Tailwind, but would using `lg:overflow-visible` here work for you in this scenario?

If you have a specific use case where you need the value to be `inherit`, you will need to add your own utility.

**Reply by GorvGoyl** [reply]
> you will need to add your own utility.  

That's what I thought but I'm not able use any custom tw utility in a SCSS file :(
See my question on SO for more details:
https://stackoverflow.com/questions/66639623/not-able-to-use-custom-classes-in-apply-in-tailwind-nextjs-project

**Reply by GorvGoyl** [reply]
@simonswiss any update on this. even the SO question didn't get any answer.. If you could confirm if it's actually a bug then I'd request to file it. (i.e. not able use any custom tw utility in a SCSS file)

**Reply by adamwathan** [reply]
Looks like it's a scoping thing related to CSS modules maybe. Add the custom classes in your config using a plugin instead:

```js
// tailwind.config.js
module.exports = {
  // ...
  plugins: [
    function ({ addUtilities }) {
      addUtilities({
        '.overflow-initial': { overflow: 'initial' },
        '.overflow-inherit': { overflow: 'inherit' },
      })
    }
  ]
}
```

**Reply by GorvGoyl** [reply]
Thank you Adam, it seems like an overkill (tinkering with tw config) but it worked. I hope a simpler solution would exist in the future.

**Comment by sylvaincodes** [comment]
//put this to globals.css 

@layer utilities {
    .overflow-inherit{
      overflow: inherit
    }
  .overflow-initial{
      overflow: initial
    }
}
